renovate.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": ["config:base", ":gitSignOff"],
```

Comment:
gitSignOff should ensure we get the sign off on each commit so that they can actually be merged